packages/core/src/renderTexture/RenderTexturePool.js

```diff
@@ -93,18 +96,20 @@ export default class RenderTexturePool
     }
 
     /**
-     * Gets extra texture of the same size as current renderTexture
+     * Gets extra texture of the same size as input renderTexture
+     *
+     * `getFilterTexture(input, 0.5)` or `getFilterTexture(0.5, input)`
      *
-     * @param {number} [resolution] resolution, by default its the same as in current renderTexture
+     * @param {PIXI.RenderTexture} input renderTexture from which size and resolution will be copied
+     * @param {number} [resolution] override resolution of the renderTexture
+     *  It overrides, it does not multiply
      * @returns {PIXI.RenderTexture}
      */
-    getFilterTexture(resolution)
+    getFilterTexture(input, resolution)
```

Comment:
I think we should rename input to something that makes it easy to know its a renderTexture `targetRenderTexture` ?

Reply:
its `input` in filters. Name is the same as in https://github.com/pixijs/pixi.js/blob/dev/packages/core/src/filters/Filter.js#L215 . Its supposed to be called from apply()